SSKeychain/SSKeychainQuery.h

```diff
@@ -75,6 +80,12 @@ typedef NS_ENUM(NSUInteger, SSKeychainQuerySynchronizationMode) {
  */
 @property (nonatomic, copy) NSString *password;
 
+/**
+ When enabled, the Filesystem Based Keychain (Legacy!) will be used.
+ By default macOS will query the new DataProtection Keychain
+ */
+@property (nonatomic, assign) BOOL legacyKeychainMode;
```

Comment:
I might rename this to `useLegacyKeychain`

Reply:
Honestly, though, I wonder if we should invert this, to call it `useModernKeychain` or something, and have it enabled by default.